### PR TITLE
fix configmap name hardcode

### LINKED ISSUE #108: Custom teardown script doesn't work
I've tried to set a custom teardown script using Helm values:

```yaml
teardown: |-
  #!/bin/sh
  path=$1
  archived_path="$(dirname ${path})/archived-$(basename ${path})"
  mv ${path} ${archived_path}
```

Although the config map gets updated to the new teardown script, when I delete a pvc local-path-provisioner still deletes the pv folder instead of running the script.

Any help would be appreciated :-)

### PR DESCRIPTION
**Pull request by nicktming**
Fix configmap issue in v0.0.15. #108 #115

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -87,6 +88,11 @@ func StartCmd() cli.Command {
 				Usage: "Paths to a kubeconfig. Only required when it is out-of-cluster.",
 				Value: "",
 			},
+			cli.StringFlag{
+				Name:  FlagConfigMapName,
+				Usage: "Paths to a kubeconfig. Only required when it is out-of-cluster.",
```
Comment (yasker) [inline main.go:93]: Wrong usage.

Comment (nicktming) [reply main.go]: ok, it is done.

Comment (nicktming) [reply main.go]: and i think i need to update the helm chart and after that you can review.

Comment (nicktming) [reply main.go]: it works for me using helm deployment now.

**Pull request merged**